src/dotnet-scaffolding/dotnet-scaffold/Telemetry/CommandExecuteTelemetryEvent.cs

```diff
@@ -15,7 +15,7 @@ public CommandExecuteTelemetryEvent(DotNetToolInfo dotnetToolInfo, CommandInfo c
         SetProperty("Version", dotnetToolInfo.Version);
         SetProperty("ToolName", dotnetToolInfo.Command);
         SetProperty("ToolLevel", dotnetToolInfo.IsGlobalTool ? TelemetryConstants.GlobalTool : TelemetryConstants.LocalTool);
-        SetProperty("CommandName", commandInfo.Name);
+        SetProperty("CommandName", commandInfo.Name.Hash());
         SetProperty("AllCommandCategories", string.Join(",", commandInfo.DisplayCategories).Hash());
```

Comment:
Not new here, but we probably want to Hash each category, not the comma-delimeted list. Otherwise our well-known values list becomes really complicated.

Reply:
yep adding an extension for that, will work differently than the isPII scenario, verified